dts/x86/intel/raptor_lake_s.dtsi

```diff
@@ -237,6 +237,14 @@
 			clock-frequency = <100000000>;
 			interrupts = <PCIE_IRQ_DETECT IRQ_TYPE_LOWEST_LEVEL_LOW 3>;
 			interrupt-parent = <&intc>;
+
+			status = "okay";
+		};
+
+		spi0_dma: spi0_dma {
+			compatible = "intel,lpss";
+			dma-parent = <&spi0>;
```

Comment:
dma-parent need to be removed and dmas need to be added in spi0 instance, same applies for all instances.